container/squid/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM ubuntu/squid:edge
```

Comment:
This is not used, my previous comment does suggest using it.

Reply:
@algchoo this comment still stands. The values file is pointing to `ubuntu/squid:edge` instead of using this image.